model/Plan.js

```diff
@@ -5,13 +5,16 @@ const Schema = mongoose.Schema;
     This model refers to a user plan.
     A user can have multiple plans for different combination of majors.
 */
-const planSchema = new Schema({
-  name: { type: String, required: true },
-  majors: { type: [String] },
-  year_ids: [{ type: Schema.Types.ObjectId, ref: 'Year' }],
-  distribution_ids: [{ type: Schema.Types.ObjectId, ref: 'Distribution' }],
-  user_id: { type: String, required: true },
-  expireAt: { type: Date, expires: 60 * 60 * 24 },
-});
+const planSchema = new Schema(
+  {
+    name: { type: String, required: true },
+    majors: { type: [String] },
+    year_ids: [{ type: Schema.Types.ObjectId, ref: 'Year' }],
+    distribution_ids: [{ type: Schema.Types.ObjectId, ref: 'Distribution' }],
+    user_id: { type: String, required: true },
+    expireAt: { type: Date, expires: 60 * 60 * 24 },
+  },
+  { timestamps: true },
```

Comment:
oh sick! sheesh! 

but does this add createdAt and updatedAt fields to documents already in mongodb?

Reply:
It doesn't, but when you update an existing document, it does add the updatedAt field it seems. In the frontend version of pr, I added code to deal with the "old" plans that will still have the updatedAt field as null (they're considrred least recently updated because the updatedAt field should be non-null once users update the existing document)..

Might be wrong about this, but for where to sort, I dont think we have a backend data structure/scheme that stores a reviewer and all the plans they have access to review? Also, I'd assume  no reviewer is gonna realistically have more than ~100 current reviewees(prolly a lot less) so even a frontend sort should be trivially fast(and all the other reviewee sorts happen in the frontend)?